frontend/src/components/data-analytics/FilterSidebar.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from "react";
+import CheckBox from "./Checkbox";
+interface AppProps {}
+
+interface ArrObject {
+  id: number;
+  value: string;
+  isChecked: boolean;
+}
+
+interface AppState {
+  age: ArrObject[];
+}
+
```

Comment:
when the side bar is initially created, all checkboxes should be checked off. 

<img width="284" alt="image" src="https://github.com/cornellh4i/okb-hope/assets/140224336/eccadd70-6544-45fe-b57d-4005501d33c2">